Infrastructure/stack/eip.tf

```diff
@@ -1,6 +1,6 @@
 # Module for Creating Elastic Ip address.
 module "harshvardhan-eip" {
-  source  = "./modules/eip"
+  source  = "../modules/eip"
```

Comment:
Dont need to create a separate file for every resource, you can include this in bastion.tf/nat.tf